### DISCUSSION TITLE
installing Seurat in RStudio

### DISCUSSION THREAD

**Discussion by sbehar1961** [satijalab/seurat#4867] [Q&A] [answered]
I am interested in learning Seurat, but didn't get very far. I don't seem to be able to install Seurat in RStudio. Am I doing something wrong or is there a work around?
Thanks,
Sam Behar
UMass Medical School

FROM RSTUDIO:

The downloaded binary packages are in
	/var/folders/d0/gn7687dj1pdcllbblp2ht2700000gp/T//RtmpYG6zoV/downloaded_packages
> install.packages('Seurat')
Warning in install.packages :
  package ‘Seurat’ is not available (for R version 3.6.2)

**Comment by timoast** [comment, marked as answer]
Seurat currently requires R version >= 4.0.0

**Reply by 1Sage1** [reply]
Updating R solved this problem for me. Why don't you try that?
